sdks/java/io/cassandra/src/main/java/org/apache/beam/sdk/io/cassandra/CassandraIO.java

```diff
@@ -197,6 +196,12 @@ private CassandraIO() {}
       return builder().setConsistencyLevel(consistencyLevel).build();
     }
 
+    public Read<T> withMinNumberOfSplits(Integer minNumberOfSplits) {
```

Comment:
Can you please document this method and explain that this optional and what is the intended behavior.